School.js

```diff
@@ -1,14 +1,19 @@
-const Class = require('./Class');
-const Student = require('./Student')
+
+
+const Class = require('./Class.js');
+const Student = require('./Student.js')
 
 class School {
   constructor() {
     this.classes = {
+      // Math:{"teacher": "John Doe", "students":[{name:"Kim",age:12,city:"CHI",grade:"fail"},{name:"Rachel",age:17,city:"NYC",grade:"pass"}]},
```

Comment:
I typically have one of my last commits be a `cleanup` commit where i delete commented out code